pdu/PDUHeader.go

```diff
@@ -76,9 +69,5 @@ func (c *Header) SetSequenceNumber(v int32) {
 
 // Marshal to buffer.
 func (c *Header) Marshal(b *ByteBuffer) {
-	b.Grow(16)
-	b.WriteInt(c.CommandLength)
-	b.WriteInt(int32(c.CommandID))
-	b.WriteInt(int32(c.CommandStatus))
-	b.WriteInt(c.SequenceNumber)
+	_ = binary.Write(b, binary.BigEndian, c)
```

Comment:
What happen if we add another field for internal metadata. They will be marshaled to buffer? I don't think we should do that.